ws-nextjs-app/pages/[service]/send/[id]/ClosedScreen/index.tsx

```diff
@@ -0,0 +1,58 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useContext, useEffect } from 'react';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import TimeStampContainer from '#app/legacy/psammead/psammead-timestamp-container/src';
+import GenericMessage from '../GenericMessage';
+import styles from './styles';
+import fallbackTranslations from '../fallbackTranslations';
+
+type Props = {
+  title: string;
+  closingTime?: string;
+};
+
+export default function ClosedScreen({ title, closingTime }: Props) {
+  const {
+    timezone,
+    locale,
+    altCalendar,
+    service,
+    script,
+    translations: {
+      ugc: {
+        closedHeading = fallbackTranslations.closedHeading,
+        closedDescription = fallbackTranslations.closedDescription,
+      } = {},
+    },
+  } = useContext(ServiceContext);
+
+  useEffect(() => {
+    document.title = `${closedHeading}: ${title}`;
+  }, [closedHeading, title]);
```

Comment:
Is there a reason why this title changes has to be done client side?

Reply:
Good point, in this case it doesn't need to be client side. I'll update!